src/imagetextalternative.js

```diff
@@ -98,10 +95,14 @@ export default class ImageTextAlternative extends Plugin {
 	 */
 	_createBalloonPanel() {
 		const editor = this.editor;
+		const command = editor.commands.get( 'imageTextAlternative' );
 
 		const panel = new ImageBalloonPanel( editor );
 		const form = new TextAlternativeFormView( editor.locale );
 
+		form.labeledInput.inputView.bind( 'isReadOnly' ).to( command, 'isEnabled', value => !value );
+		form.saveButtonView.bind( 'isEnabled' ).to( command );
```

Comment:
Shouldn't it rather be `form.bind( 'isEnabled' ).to( command );` and then, inside `TextAlternativeFormView`, `isEnabled` should be propagated among the inputs and buttons?